armadillo/src/test/java/org/molgenis/armadillo/controller/AdminControllerTest.java

```diff
@@ -39,20 +43,32 @@
 @ActiveProfiles("test")
 @Import({AuditEventPublisher.class})
 class AdminControllerTest {
+
   public static final String EXAMPLE_SETTINGS =
```

Comment:
is this one of needed than?

Reply:
Yes it was still used by a test that asserts the output of the "get all metadata" endpoint